tests/test_modeling_common.py

```diff
@@ -3000,71 +2995,6 @@ def test_inputs_embeds_matches_input_ids(self):
                     )[0]
             self.assertTrue(torch.allclose(out_embeds, out_ids))
 
-    def test_inputs_embeds_matches_input_ids_with_generate(self):
```

Comment:
We already had a test that checks that generating from `inputs_embeds` is equivalent to generating from `input_ids` -- `test_generate_from_inputs_embeds`

This test was deleted.

Reply:
one q: afaik the other test is skipped for encoder-decoder models? Can we enable it just to be sure we cover all models with generation abilities?

Reply:
Good point, thank you for catching that! I'll update `test_generate_from_inputs_embeds` so as to support VLMs

Reply:
✅ `test_generate_from_inputs_embeds` updated for VLMs, overwrites deleted. @zucchini-nlp have a look at the updated test -- you might need to add VLMs to the list of added special cases in the test